datadog_checks_base/datadog_checks/stubs/aggregator.py

```diff
@@ -46,6 +46,21 @@ def events(self):
         """
         return self._events
 
+
+    def assert_metric_tag(self, metric_name, tag, count=None, at_least=1):
```

Comment:
can we rename this to something like `assert_metric_has_tag` and add a docstring?